core/templates/dev/head/pages/exploration_editor/ExplorationWarningsService.js

```diff
@@ -16,16 +16,22 @@
  * @fileoverview A service that lists all the exploration warnings.
  */
 
+// When an unresolved answer's frequency exceeds this threshold, an exploration
+// will be blocked from publishing until it gets resolved.
```

Comment:
... from being published until the answer is resolved.

Reply:
Done.